deploy/rp-production-parameters.json

```diff
@@ -20,6 +20,9 @@
         "armClientId": {
             "value": ""
         },
+        "azSecPackVSATenantId": {
+            "value": ""
+        },
```

Comment:
Comment on the config file not matching parameter causing golang unit test to fail.